probe/docker/container.go

```diff
@@ -220,6 +222,48 @@ func (c *container) ports(localAddrs []net.IP) report.StringSet {
 	return report.MakeStringSet(ports...)
 }
 
+func (c *container) cpuPercentMetric() report.Metric {
+	result := report.MakeMetric()
+	if len(c.pendingStats) < 2 {
```

Comment:
Ah, I guess because it's all in deltas, we don't actually know it without 2 samples. blah.